### PR TITLE
Nanopi Neo Air enable BT

### PR DESCRIPTION
**Pull request by spanceac**

### CODE REVIEW

**Comment by spanceac** [comment]
@resin-ci retest this please

**Comment by floion** [comment]
@spanceac you can add the balena-devices-team group alias for review

File: layers/meta-resin-allwinner/recipes-bsp/brcm-patchram-plus/brcm-patchram-plus.bb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+DESCRIPTION = ""
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
+
+SRC_URI = "git://github.com/balena-os/brcm_patchram_plus"
```
Comment (floion) [inline layers/meta-resin-allwinner/recipes-bsp/brcm-patchram-plus/brcm-patchram-plus.bb]: We should switch to this one: https://github.com/balena-os/meta-artik710/blob/e8383b7566ebbff9bdf996b5634613469ba845ec/recipes-bsp/broadcom-bluetooth/brcm-patchram-plus_git.bb#L10

**Comment by floion** [comment]
@resin-jenkins test this please

**Pull request merged**